test/cli/kit/table_formatter_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+
+module CLI
+  module Kit
+    class TableFormatterTest < MiniTest::Test
+      def setup
+        @tf = TableFormatter.new(:pid, :ppid, :cmd, default: [:pid, :ppid])
```

Comment:
You could recommend `TableFormatter.new(*@tfs.members, default: [:pid, :ppid])`, to avoid repeating the field names